### PR TITLE
Create Button

### LINKED ISSUE #25: As developer i want to create an action button component, so that users can navigate to different sites of the app
- The button component can have different labels.
- The button component can have different links
button3
button1
button2

### PR DESCRIPTION
**Pull request by dimas1976**
fixes #25 
https://mentalika-button-voindow33imbp.herokuapp.com/storybook/?path=/story/component-actionbutton--book-button

### CODE REVIEW

File: src/app/components/ActionButton/ActionButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import ActionButton from './ActionButton';
+import Button from './ActionButton';
+
+export default {
+  title: 'Component/ActionButton',
+  component: ActionButton,
+};
+
+export const BookButton = (): JSX.Element => (
+  <Button onClickHandle={() => console.log('huhu')}>buchen</Button>
```
Comment (marwinburesch) [inline src/app/components/ActionButton/ActionButton.stories.tsx:11]: 👋

File: src/app/components/ActionButton/ActionButton.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import type { ReactNode } from 'react';
+import styles from './ActionButton.module.css';
+
+type ActionButtonProps = {
+  children: ReactNode;
+  onClickHandle?: () => void;
```
Comment (marwinburesch) [inline src/app/components/ActionButton/ActionButton.tsx:7]: `onClick` is better and then, once you use it, name the assigned function `handleClick`

**Review by marwinburesch — COMMENTED** [review]
LGTM

File: src/app/components/ActionButton/ActionButton.tsx (hunk 2 above)
Comment (dimas1976) [reply src/app/components/ActionButton/ActionButton.tsx:7]: Done so

**Pull request merged**